### PR TITLE
[DOCS] Update nxos installation docs to use python3

### LINKED ISSUE #59750: [DOCS] Cisco Nexus Salt Minion Installation and Configuration Guide uses Python 2 for installing GuestShell Minion
**Description**
The Cisco Nexus Salt Minion Installation and Configuration Guide uses Python 2 to install the guestshell minion, which is no longer supported. Instructions need to be updated to use Python 3

**Suggested Fix**
Update documentation to make reference to Python 3 (commands and repos), not Python 2

**Type of documentation**
Cisco Nexus Salt Minion Installation and Configuration Guide

**Location or format of documentation**
https://docs.saltproject.io/en/latest/topics/installation/nxos.html#step-3-install-saltstack-minion

### PR DESCRIPTION
**Pull request by ggiesen**
### What does this PR do?
Updates the NX-OS guestshell installation instructions to install Salt using Python3

### What issues does this PR fix or reference?
Fixes: #59750

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [x] Docs

### Commits signed with GPG?
No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Review by dmurphy18 — DISMISSED** [review]

**Comment by Ch3LL** [comment]
Is this still a WIP?

**Comment by ggiesen** [comment]
> 
> 
> Is this still a WIP?

@Ch3LL I was stalling on not having access to a Nexus 3k to test for, and that won't change any time soon, but given how out of date the doc currently is, I think it's better to merge this even if it's not 100% complete rather than let the currently completely out of data doc stand. Any thoughts @dmurphy18?

**Comment by dmurphy18** [comment]
Update the docs, better than what is there

**Review by Ch3LL — DISMISSED** [review]

**Comment by ggiesen** [comment]
re-run pr-arch-lts-x86_64-py3-pytest

File: doc/topics/installation/nxos.rst [hunk 1]
```diff
@@ -257,24 +257,30 @@ The ``guestshell`` is an independent CentOS container that does not inherit sett
...
 
 Install the ``certifi`` python package.
 
-  ``[root@guestshell guestshell]# pip install certifi``
+  ``[root@guestshell guestshell]# pip3 install certifi``
 
-The most current information on installing the SaltStack Minion in a Centos7 environment can be found here_
+The most current information on installing the SaltStack Minion in a CentOS 7 environment can be found here_
 
 .. _here: https://repo.saltstack.com/#rhel
```
Comment (s0undt3ch) [inline doc/topics/installation/nxos.rst:274]:
```suggestion
.. _here: https://repo.saltproject.io/#rhel
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Review by garethgreenaway — DISMISSED** [review]

**Review by ScriptAutomate — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Pull request merged**